[0m11:58:06.302379 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B34A35CF20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B34CC8A750>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B34CC8A180>]}


============================== 11:58:06.312554 | 308ba73e-58fa-470a-8d8f-e482b4b017d0 ==============================
[0m11:58:06.312554 [info ] [MainThread]: Running with dbt=1.9.4
[0m11:58:06.315576 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\HP\\.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt run', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m11:58:06.317589 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\MSDA\semester 2\Date 226_DW\Lab2_ETL\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:58:06.317589 [debug] [MainThread]: Command `dbt run` failed at 11:58:06.317589 after 0.31 seconds
[0m11:58:06.319722 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B34CAB5E80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B34A35CF20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B34D17E8D0>]}
[0m11:58:06.320729 [debug] [MainThread]: Flushing usage events
[0m11:58:06.834532 [debug] [MainThread]: An error was encountered while trying to flush usage events
